src/test/java/teammates/test/cases/BaseTestCase.java

```diff
@@ -116,23 +116,35 @@ protected static void ignorePossibleException() {
         assertTrue(true);
     }
     
+    /**
+     * Invokes the method named {@code methodName} as defined in the {@code definingClass}.
+     * @param definingClass     the class which defines the method
+     * @param methodName
+     * @param parameterTypes    the parameter types of the method,
+     *                          which must be passed in the same order defined in the method
+     * @param invokingObject    the object which invokes the method, can be {@code null} if the method is static
+     * @param args              the arguments to be passed to the method invocation
+     */
+    protected static Object invokeMethod(Class<?> definingClass, String methodName, Class<?>[] parameterTypes,
+                                         Object invokingObject, Object[] args)
+            throws Exception {
```

Comment:
I don't really like the 2nd one. It's squishing everything to the right (so many leading whitespaces). The 1st one is better, but also it works only because the line is as short as `throws Exception`. The moment we have `throws SomeLongExceptionA, SomeLongExceptionB`, neither of the alternatives will work. Example can be seen in `Logic#createInstructorAccount`:
```java
public void createInstructorAccount(String googleId, String courseId, String name, String email,
                                    Boolean isArchived, String roleParam, boolean isDisplayedToStudents,
                                    String displayedNameParam, String privileges, String institute)
        throws EntityAlreadyExistsException, InvalidParametersException {
    ...
}
```